### ISSUE TITLE
APM needs to be re-compiled against Node.js 7

### ISSUE THREAD

**Issue by fusion809** [tensor5/arch-atom#42]
Hi,

Whenever I run `apm update` I get the error message:

```bash
module.js:600
  return process.dlopen(module, path._makeLong(filename));
                 ^

Error: The module '/usr/lib/node_modules/atom-package-manager/node_modules/git-utils/build/Release/git.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 48. This version of Node.js requires
NODE_MODULE_VERSION 51. Please try re-compiling or re-installing
the module (for instance, using `npm rebuild` or`npm install`).
    at Object.Module._extensions..node (module.js:600:18)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/lib/node_modules/atom-package-manager/node_modules/git-utils/lib/git.js:8:16)
    at Object.<anonymous> (/usr/lib/node_modules/atom-package-manager/node_modules/git-utils/lib/git.js:371:4)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
```

this error message seems to me to be saying you need to re-compile APM against the new Node.js. So please bump APM's pkgrel to get the Arch Linux build servers to do the required rebuild.

Thanks for your time,
Brenton

**Comment by tensor5** [comment]
Thanks. I reported it the nodejs maintener https://bugs.archlinux.org/task/51628.

**Comment by tensor5** [comment]
@Xananax, can you tell me what is the apm command that triggered the error and the exact output?

**Comment by tensor5** [comment]
BTW, the command to update atom packages from the command line is `apm update`.

**Comment by tensor5** [comment]
What version of nodejs are you running?

**Comment by Xananax** [comment]
> If I understand correctly you installed atom from source, without building it form this PKGBUILD

I used the default package. However, 

> What version of nodejs are you running?

I think you put your finger on it; I don't have the regular `nodejs` package, I am using, for some reason [nodejs-lts-carbon](https://www.archlinux.org/packages/community/x86_64/nodejs-lts-carbon/). I have no recollection of this, some package must've needed it and I switched them without thinking.

I confirm that if I uninstall `pacman -R nodejs-lts-carbon` and reinstal `pacman -S nodejs`, Apm & Atom both work. The problem was on my end, I'm truly sorry for the bother and the waste of time.

Thanks for the support!
